### ISSUE TITLE
Deno language server could not find .mjs module

### ISSUE THREAD

**Issue by yubaoquan** [denoland/vscode_deno#239]
### test

**Deno language server could not find .mjs module**

**To Reproduce**

1. Create a file xxx/a.mjs 
```javascript
export default function() {
  console.info('hello');
}
```
2. Create a file b.mjs
```javascript
import fn from './xxx/a.mjs';

fn()
```
3. Look

**Expected behavior**

Ctrl+click the import statement in b.mjs will jump to a.mjs

**Screenshots**

![image](https://user-images.githubusercontent.com/2340296/95537250-ccbaf300-0a1f-11eb-8280-21e0adedef2b.png)

**Versions**

Version: 1.50.0 (user setup)
Commit: 93c2f0fbf16c5a4b10e4d5f89737d9c2c25488a3
Date: 2020-10-07T06:10:52.432Z
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.18362

deno: 1.4.2

extension: 2.3.1

**Comment by kitsonk** [comment]
I can no longer reproduce this with Deno 1.8.3 and extension version 3.2.0.

There were some changes to properly handle extensions like `.mjs` properly in the recent past and I suspect they resolved the issue.

**Comment by yubaoquan** [comment]
Also fixed in deno 1.8.2 and extension version 3.2.0.
